lib/puppet/face/module/list.rb

```diff
@@ -140,7 +140,7 @@ def warn_unmet_dependencies(environment)
             parent_version     = dep[:parent][:version]
 
             msg = "'#{parent_name}' (#{parent_version})"
-            msg << " requires '#{dep_name}' (#{version_constraint})"
+            msg << _(" requires '%{dep_name}' (%{version_constraint})") % { dep_name: dep_name, version_constraint: version_constraint }
```

Comment:
Do you think we could just put that all in one string? I can't see any reason for it to be split up, and it may be untranslatable this way.